include/bluetooth/gatt.h

```diff
@@ -1344,8 +1343,16 @@ struct bt_gatt_read_params {
 			/** Attribute data offset. */
 			uint16_t offset;
 		} single;
-		/** Handles to read in Read Multiple Characteristic Values. */
-		uint16_t *handles;
+		struct {
+			/** Attribute handles to read with Read Multiple
+			 *  Characteristic Values.
+			 */
+			uint16_t *handles;
+			/** If true, we expect that the read characteristics
```

Comment:
change wording to use the term variable length.

It is actually a bit more complicated.
The READ_MULT operation can be used for values with fixed and known lengths, except for the last value which can be variable.
The READ_MULT_VL operation can be used for values with variable or unknown length.

I think it would be best that we describe this option as more clear on what we will do with this flag.

Suggested wording:
```
/** If true use Read Multiple Variable Length Characteristic Values procedure. 
 *  The values of the set of attributes may be of variable or unknown length.
 *  If false use Read Multiple Characteristic Values procedure.
 *  The values of the set of attributes must be of a known fixed length, with the
 *  exception of the last value that can have a variable length.
 */